packages/web-components/src/components/va-minimal-header/va-minimal-header.scss

```diff
@@ -0,0 +1,35 @@
+@forward 'settings';
```

Comment:
Does this need to be done? I think it's only needed for USWDS/v3 specific components.

Reply:
Right, this was a remnant for earlier, removed.